### PR TITLE
Ungrouped actions

### PR DESCRIPTION
**Pull request by bergie**
Originally we used brackets to identify the action of a packet. This PR switches us to use the `action` payload instead.

Should enable us to remove setTimeouts from action dispatching, as well as remove some legacy dependencies.

### CODE REVIEW

**Comment by bergie** [comment]
3d1159d8270a77c3bfc60512577ff10744c82be3 demonstrates the missing piece. We have some reducers that have to handle multiple-level wildcards, so `foo:*` and `foo:*:*`

File: components/DispatchAction.coffee [hunk 1]
```diff
@@ -26,29 +26,25 @@ exports.getComponent = ->
   c.outPorts.add 'handle',
     datatype: 'all'
     addressable: true
+  c.outPorts.add 'handling',
+    datatype: 'integer'
```
Comment (jonnor) [inline components/DispatchAction.coffee]: What is this for? Could use a description

**Comment by jonnor** [comment]
Looks good

File: components/DispatchAction.coffee (hunk 1 above)
Comment (bergie) [reply components/DispatchAction.coffee]: This is where we send the chosen handle index. Mostly just for feature-compatibility vs. routers/GroupRouter

**Pull request merged**